packages/nimbus/src/components/data-table/constants.tsx

```diff
@@ -0,0 +1,4 @@
+export const UPDATE_ACTIONS = {
```

Comment:
This file is likely going to grow into a bunch of settings constants for managing table configurations to decide which action would be triggered.